### PR TITLE
BaSP-TG-41: Redux Setup

### PR DESCRIPTION
**Pull request by LucianaPrat**
**Carrying out the Setup to use REDUX:**

- Install redux
- Install redux thunk
- Implement provider
- Create and configure Store
- Create a module for each resource with constants, actions, reducer and thunks
- Setup React-redux-tools

### CODE REVIEW

File: src/redux/Admins/reducer.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { GET_ADMINS_ERROR, GET_ADMINS_PENDING, GET_ADMINS_SUCCESS } from './types';
+
+const INITIAL_STATE = {
+  list: [],
+  pending: false,
```
Comment (Baldizan) [inline src/redux/Admins/reducer.js:5]: maybe isPending would be more descriptive

File: src/redux/Admins/thunks.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { getTasksError, getTasksPending, getTasksSuccess } from './action';
```
Comment (elias-92) [inline src/redux/Admins/thunks.js:1]: you copied the thunks from task, it should be admins.

**Review by elias-92 — COMMENTED** [review]
comment

File: src/redux/Admins/actions.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { GET_TASKS_PENDING, GET_TASKS_SUCCESS, GET_TASKS_ERROR } from './types';
```
Comment (Baldizan) [inline src/redux/Admins/actions.js]: this is the admins file

File: src/redux/Admins/thunks.js (hunk 2 above)
Comment (Baldizan) [inline src/redux/Admins/thunks.js]: this is the admins file

File: src/redux/TimeSheets/reducer.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import { GET_TIMESHEETS_ERROR, GET_TIMESHEETS_PENDING, GET_TIMESHEETS_SUCCESS } from './types';
+
+const INITIAL_STATE = {
+  list: [],
+  pending: false,
```
Comment (Baldizan) [inline src/redux/TimeSheets/reducer.js:5]: this should be "isPending"

**Review by Baldizan — APPROVED** [review]

**Review by elias-92 — APPROVED** [review]

**Review by nailaghigo — APPROVED** [review]

**Pull request merged**